automation-tests/test/environmentSettings.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
```

Comment:
2023 for all these new files now!

Reply:
They were created in 2022, but I'll go update them